src/styles/GlobalStyle.ts

```diff
@@ -15,6 +14,7 @@ export const GlobalStyle = createGlobalStyle`
   
   body {
     font-size: 1.6rem;
+    color: ${props => props.theme.text_primary_color};
     background-color: ${props => props.theme.background_primary};
```

Comment:
```suggestion
    color: ${({ theme }) => theme.color.text_primary_color};
    background-color: ${({ theme }) => theme.color.background_primary};
```
위와 같이 수정이 가능할 거 같습니다.

현재 충돌이 나고 있는 상황인 거 같은데 dev 참고하셔서 수정해주시면 될 거 같습니다!
본인 feature 작업 재개 전에 항상 dev에서 pull 당겨주시는 습관 들이시면 충돌 발생하는 것을 초기에 잡을 수 있습니다!